src/pages/ToDoList/List.js

```diff
@@ -1,34 +1,78 @@
-import React from 'react';
+import React, {useState, useMemo} from 'react';
```

Comment:
Import `useEffect` here, so you can write `useEffect` instead of `React.useEffect`:
```suggestion
import React, { useState, useMemo, useEffect } from 'react';
```